docker-compose.yml

```diff
@@ -39,25 +39,22 @@ services:
         awslogs-stream: "elasticsearch"
 
   migration: 
-    build:
-      context: .
-      dockerfile: config/Dockerfile_webservice
+    image: quay.io/dockstore-testing/dockstore-webservice:${DOCKSTORE_VERSION}
```

Comment:
We should probably get a proper `dockstore` organization for these

Reply:
Actually, we have one https://quay.io/organization/dockstore. We should get it moved and a proper gravatar setup @agduncan94

Reply:
Can you please add me to the quay.io dockstore org?